dx.yml

```diff
@@ -8,6 +8,9 @@ package:
         outdir: "out/dxos-templates"
 runtime:
   client:
+    remoteSource: https://halo.dev.dxos.org/vault.html
```

Comment:
why halo dev by default?

Reply:
I'll go ahead and merge pointing to dev for now. Once we release, I'll move it back to prod.

Reply:
switched to use the dev version of HALO, can revert that bit once https://github.com/dxos/dxos/pull/4611 (and future fixes?) make it to prod